core/integration/container_test.go

```diff
@@ -2606,11 +2606,6 @@ func TestContainerExport(t *testing.T) {
 		require.NoError(t, err)
 		require.True(t, ok)
 
-		stat, err := os.Stat(imagePath)
-		require.NoError(t, err)
-		require.NotZero(t, stat.Size())
-		require.EqualValues(t, 0o600, stat.Mode().Perm())
-
```

Comment:
Looks like some of the changes from https://github.com/dagger/dagger/pull/5649 got dropped too

Reply:
wtf... I literally just `git pull -r upstream main`'d... maybe I'll just `git reflog` my way back and try again 🤷‍♂️

Reply:
Thank you for catching all this though!